tests/fdb/test_fdb_mac_learning.py

```diff
@@ -199,18 +224,23 @@ def testFdbMacLearning(self, ptfadapter, duthosts, rand_one_dut_hostname, ptfhos
 
         # unshut 1 port and populate fdb for that port. make sure fdb entry is populated in mac table
         duthost = duthosts[rand_one_dut_hostname]
-        duthost.shell("sudo config interface startup {}".format(target_ports_to_ptf_mapping[0][0]))
-        time.sleep(30)
+        target_ports = [target_ports_to_ptf_mapping[0][0]]
+        duthost.shell("sudo config interface startup {}".format(target_ports[0]))
+        pytest_assert(wait_until(150, 5, 0, self.check_mux_status_consistency, duthost, target_ports))
```

Comment:
do we need to check for dualtor first? What if this is a t0?

Reply:
Thanks @lolyu for catching this, yeah for `t0` mux status is irrelevant (as muxcable is specific to dualtor), will update `check_mux_status_consistency` method to handle this case.

Reply:
Hi @lolyu I have updated the fix to take care of non-dualtor topologies, please review.